packages/amplify-category-function/provider-utils/awscloudformation/cloudformation-templates/lambda-cloudformation-template.json.ejs

```diff
@@ -214,6 +222,35 @@
         }
         <% } %>
         <% } %>
+        <% if (props.cloudwatchEvent && props.cloudwatchEvent != "NONE") { %> 
+        ,"CloudWatchEvent": {
+            "Type" : "AWS::Events::Rule",
+            "Properties":{
+                "Description" : "Schedule rule for Lambda",
+                "ScheduleExpression" : "<%= props.cloudwatchRule %>",
```

Comment:
Here the CF parameter should be referenced with `Ref` not the ejs template property